### ISSUE TITLE
All ευ being in Preroot to move to the Base1 position.

### ISSUE THREAD

**Issue by Hippodamie** [eXistSolutions/LGPN-ling#518]
Dear Magdalena,

1. Could you, please, move all the ευ which are currently en Preroot level into Base1?

It is a definitive change we need.

2. And could you also definitely change the current title Preroot into Prefix, as it was at the beginning?

Many thanks in advance.
Have a good next week,
with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,

could you, please, make this issue the priority? Because, it is easy (for us): we do not have to check. 

And wait more for the 510# (εσ) parentheses to delete, since we do not have time to check for it right now.

If we do not have contact, I wish you and all your family a merry Christmas!
Take a break I hope. Have nice holidays.
We (LGPN-Ling) will not work a lot online until January the 7th.

With all best wishes and thanks for all,
Sophie

**Comment by tuurma** [comment]
Working on it today, sorry, I've been unwell past days.

**Comment by tuurma** [comment]
@Hippodamie what shall I do with ca 30 cases (of 727 ευ names in total), where there's already something in Base1, like for Εὐθουμοκλεῖς

**Comment by Hippodamie** [comment]
Very well! Thanks !
Could you please list the ca 30 cases? It means we will deal with those, and not you. But it will be great to have their list...
But if you do the job for all the other, it will be great!!!
Many thanks.
Best,
Sophie

**Comment by tuurma** [comment]
List of the names to investigate individually is at http://clas-lgpn4.classics.ox.ac.uk:8080/exist/apps/lgpn-ling/import/euNames.xq

**Comment by Hippodamie** [comment]
Wonderful!
Thanks a lot!
Best,
Sophie

**Comment by tuurma** [comment]
updated other names with eu prefix with

```xquery
xquery version "3.1";

declare namespace tei="http://www.tei-c.org/ns/1.0";

for $i in collection('/db/apps/lgpn-ling-data/data/names')//tei:m[.='ευ'][@type='prefix'][../tei:m[@type="radical"][@n='1'][string(.)='']]

let $prefix := <m xmlns="http://www.tei-c.org/ns/1.0" ana="" baseForm="" cert="" corresp="" function="" n="1" subtype="" type="prefix"/>
let $base1 :=  <m  xmlns="http://www.tei-c.org/ns/1.0" ana="{$i/@ana}" baseForm="{$i/@baseForm}" cert="{$i/@cert}" corresp="{$i/@corresp}" function="{$i/@function}" n="1" subtype="{$i/@subtype}" type="radical">{$i/string()}</m>


return 

    (update replace $i with $prefix,
    update replace $i/../tei:m[@type="radical"][@n='1'] with $base1)
```

**Comment by tuurma** [comment]
addressed with adf93332e9a0b224b5612e9011d7fcc4e8fb3da1
